### ISSUE TITLE
False positive with testbinary command

### ISSUE THREAD

**Issue by jfirebaugh** [mapbox/node-pre-gyp#270]
The `testbinary` command does not seem to validate that the binary actually exists.

1. Create an empty directory and add the following `package.json` file in it:
```
{
  "name": "test",
  "version": "0.0.0",
  "main": "index.js",
  "dependencies": {
    "node-pre-gyp": "^0.6.33"
  },
  "binary": {
    "module_name": "test",
    "module_path": "./lib/",
    "host": "https://mapbox-node-binary.s3.amazonaws.com"
  }
}
```
2. Run `npm install`
3. Run `./node_modules/.bin/node-pre-gyp testbinary`

Expected behavior: get an error saying that the binary doesn't exist.

Actual behavior:

```
node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.6.33
node-pre-gyp info using node@4.7.3 | darwin | x64
node-pre-gyp info validate Running test command: '/usr/local/Cellar/node@4/4.7.3/bin/node --eval 'require(\'/Users/john/Development/test-node-pre-gyp/lib/test.node\')''
node-pre-gyp info ok 
```

**Comment by springmeyer** [comment]
Thanks for the report. I can replicate. I've tracked the problem down to this commit: https://github.com/mapbox/node-pre-gyp/commit/7ed0d0860a801a331c136a4d9d73a6622471cb73. Reverting that fixes the problem.

**Comment by springmeyer** [comment]
No response from @edgarsilva - so this is reverted and fixed by #283
